Impossible-Agent-Suite/DangerousActivities/src/bsc.dangerous.ts

```diff
@@ -0,0 +1,88 @@
+export default [
```

Comment:
This filename should have `.addresses` in it similar to `impossible.addresses.ts` so when looking at the filename it's clear that this filename contains dangerous BSC addresses.

Reply:
Are the list of dangerous addresses going to stay as a static plaintext file forever? Surely this would become outdated at some point and there would be new dangerous addresses that this agent would miss.

Reply:
Sure no, we are waiting the client specification